capstone/capdb/models.py

```diff
@@ -633,32 +649,65 @@ def set_xml_checksums_need_update(self, value=True):
     def get_absolute_url(self):
         return reverse('volumemetadata-detail', args=[self.pk], scheme="https")
 
-    def set_duplicate(self, is_duplicate, duplicate_of=None):
+    def set_duplicate(self, duplicate_of):
         """
             Update this volume to reflect that it is or is not a duplicate of another volume.
             Update volume.case_metadatas.duplicate and .in_scope to match.
         """
+        #TODO: PDF renaming/deleting functionality
         with transaction.atomic(using='capdb'):
-            self.duplicate = is_duplicate
+            self.duplicate = True
             self.duplicate_of = duplicate_of
             self.save()
-            self.case_metadatas.update(duplicate=is_duplicate)
+            self.case_metadatas.update(duplicate=True)
             self.case_metadatas.update_in_scope()
+        CaseMetadata.reindex_cases(self.case_metadatas.all(), prune_duplicates=True)
 
-    def set_reporter(self, reporter):
+    def set_volume_number(self, volume_number, update_citations=True):
+        """
```

Comment:
Nit: update comment to refer to this function.

Reply:
Do'h.